Library/Homebrew/formula_creator.rb

```diff
@@ -138,7 +138,7 @@ class #{Formulary.class_s(name)} < Formula
         <% elsif @mode == :perl %>
           uses_from_macos "perl"
         <% elsif @mode == :python %>
-          depends_on "python"
+          depends_on "python@x.y"
```

Comment:
I don't think this is a good idea. It implicitly tells formula creators to pin their formulae to a specific Python version, which runs counter to basic Homebrew philosophy (use the latest dependencies unless they break this formula).